website/content/en/docs/cloud-providers/AWS.md

```diff
@@ -0,0 +1,191 @@
+---
+title: "Amazon Web Services (AWS)"
+linkTitle: "AWS"
+weight: 10
+---
+
+## Control Provisioning with Labels
+
+The [Provisioner CRD]({{< ref "provisioner-crd.md" >}}) supports defining
+instance types, node labels, and node taints.
+
+For certain well-known labels (documented below), Karpenter will provision
+nodes accordingly. For example, in response to a label of
+`topology.kubernetes.io/zone=us-east-1c`, Karpenter will provision nodes in
+that availability zone.
+
+Regarding AWS, 3 types of provisioning constraints are recognized: 
+- [Instance Type](#instance-type-allowlist)
+- [Node Labels](#node-lables-in-provisioner-spec) (e.g., Architecture, Capacity Type)
+- [Pod Labels](#pod-labels) (e.g., GPU)

```

Comment:
I think this pod label's piece isn't quite accurate. Specifically, these are pod.spec.nodeSelector.

Reply:
what is a good more general name? pod requirements?